model/database/metrics.yaml

```
@@ -133,3 +134,21 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.query.limit
+    type: metric
+    metric_name: db.query.limit
+    brief: "Max item count requested by the feed operation (ex. Query, ReadAll, ReadMany, Change Feed) "
+    instrument: histogram
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.client.minimal
+
+  - id: metric.db.query.response.item_count
+    type: metric
+    metric_name: db.query.response.item_count
+    brief: "Actual item count returned by the feed operation (ex. Query, ReadAll, ReadMany, Change Feed)"
```

Could you please update the description to be more generic? 
E.g. 

```suggestion
    brief: "The actual number of records returned by the query."
```